Add Dev Preview updates in Changelog (#2766)
ocaml/ocaml.org: data/changelog/dune/2024-09-25-call-for-feedback.md

---
title: Call for Feedback
tags: [dune, platform, developer-preview]
---

Hello folks! 👋 

> We'd like to welcome everyone to try and play with the [Dune Developer Preview](https://dune.ci.dev)! 🎉 

Reviewer: ```suggestion
We'd like to welcome everyone to try and play with the [Dune Developer Preview](https://preview.dune.build)! 🎉

This experimental nightly release of dune includes a lot of improvements, including the much expected package management features, and it can be installed from that website or by using the new installation script:

```bash

> $ curl https://dune.ci.dev/install | bash

Reviewer: ```suggestion
$ curl -fsSL https://get.dune.build/install | bash
```

* `dune build @deps` - to build all dependencies, useful for staged builds in Dockerfiles
* `dune pkg add <name>` - to make adding packages straightforward
* a short-hand syntax for pins on github
* and more!

If you've got any ideas, we'd love to hear them, so please open a feature request on Github 🙏 

## Other updates

#### FunOCaml Presentation
At **FunOCaml** we had a last-minute opportunity to present the work being done on Dune and we used it to introduce the Developer Preview to the community, and even tested Package Management live with suggestions from the audience (thanks @anmonteiro and Paul-Elliot for participating!) – you can [watch it on Twitch](https://www.twitch.tv/videos/2252408016?t=08h12m00s).

#### New design
We're working with @Claire_Vandenberghe on redesigning the Developer Preview website so that it'd feel like a seamless extension of OCaml.org – in this current iteration we've made it easier to get started and we're putting the FAQ front and center.

We'll be iterating on this design in the coming weeks until it fits perfectly within the OCaml.org design system 🎨 

> You can check the new website here: https://dune.ci.dev 

Reviewer: ```suggestion
You can check the new website here: https://preview.dune.build
```
